pipelines/templates/restore-build-publish-test.yml

```diff
@@ -38,7 +38,7 @@ steps:
 - template: run-unittests.yml
   parameters:
     name: WinGet.RestSource.UnitTest
-    source: '$(Build.SourcesDirectory)\src\WinGet.RestSource.UnitTest\bin\$(BuildConfiguration)\net5.0'
+    source: '$(Build.SourcesDirectory)\src\WinGet.RestSource.UnitTest\bin\$(BuildConfiguration)\netcoreapp3.1'
```

Comment:
What is the draw back of using 3.1 rather than 5.0?

Reply:
We have validated that 3.1 does not encounter the same runtime issues in the stock deployment delineated by the reference arm templates?

Reply:
I validated that 3.1 doesn't encounter the runtime issues that .net 5.0 was encountering  in the Azure function app. I don't see any drawbacks of using 3.1 as we are not tied to any specific functionality in .Net core that is in 5.0 and not in 3.1.